src/cloudStorage/googleCloud.utils.ts

```diff
@@ -12,10 +12,14 @@ export const uploadMusicImageFilter = (req, file, callback) => {
       );
 };
 
-export const uploadFileName = (req, file, callback) => {
+export const uploadFileName = (_req, file, callback) => {
   const fileNameSplit = file.originalname.split('.');
   const fileExt = fileNameSplit[fileNameSplit.length - 1];
-  callback(null, `${req.query.documentType}/${Date.now()}.${fileExt}`);
+  callback(null, `${Date.now()}.${fileExt}`);
+};
+
+const uploadDestination = (req, _file, callback) => {
+  callback(null, `${req.query.documentType}`);
```

Comment:
Since you mention it, how about we change this line into `callback(null, `${_file.fieldname}`);`, It will make file path look a bit better.
![image](https://user-images.githubusercontent.com/70059585/224488914-b88f9cd1-e2f8-42a8-8a4e-41a78c5ecdaa.png)

Reply:
Fixed